### ISSUE TITLE
Big Transfer (BiT): General Visual Representation Learning

### ISSUE THREAD

**Issue by josepsmartinez** [unico-labs/ml-reading-club#89]
[arXiv](https://arxiv.org/abs/1912.11370)

Outra referência do paper eleito [`How to train your ViT? Data, Augmentation, and Regularization in Vision Transformers`](https://github.com/unico-labs/ml-reading-club/issues/75).

Motivo da minha escolha: estuda o quanto a prática de transfer learning (frequente dentro do nosso time) é capaz de melhorar a performance do modelo na tarefa destino (downstream task).

Abstract:
`
Transfer of pre-trained representations improves sample efficiency and simplifies hyperparameter tuning when training deep neural networks for vision.
We revisit the paradigm of pre-training on large supervised datasets and fine-tuning the model on a target task. We scale up pre-training, and propose a simple recipe that we call Big Transfer (BiT). By combining a few carefully selected components, and transferring using a simple heuristic, we achieve strong performance on over 20 datasets.
BiT performs well across a surprisingly wide range of data regimes -- from 1 example per class to 1M total examples. BiT achieves 87.5% top-1 accuracy on ILSVRC-2012, 99.4% on CIFAR-10, and 76.3% on the 19 task Visual Task Adaptation Benchmark (VTAB).
On small datasets, BiT attains 76.8% on ILSVRC-2012 with 10 examples per class, and 97.0% on CIFAR-10 with 10 examples per class.
We conduct detailed analysis of the main components that lead to high transfer performance.
`

**Comment by josepsmartinez** [comment]
THIS
